src/components/DayPicker.jsx

```diff
@@ -96,6 +98,9 @@ const propTypes = forbidExtraProps({
   onNextMonthClick: PropTypes.func,
   onMonthChange: PropTypes.func,
   onYearChange: PropTypes.func,
+  onMultiplyScrollableMonths: PropTypes.func, // VERTICAL_SCROLLABLE daypickers only
```

Comment:
I believe since you started this PR `onMultiplyScrollableMonths` has been removed (https://github.com/airbnb/react-dates/commit/a3e76feb193d09af640e226bf33f7d98f628cad8#diff-87325f2795c2738c7a5b679475232f8e).  This doesn't look like it's being used anywhere anymore so you should probably remove it.